apps/web/src/app/api/sanity/route.ts

```diff
@@ -1,152 +1,166 @@
-import { revalidatePath } from "next/cache";
 import { NextResponse } from "next/server";
-import { eq, sql } from "drizzle-orm";
-import { z } from "zod";
+import { and, eq, inArray } from "drizzle-orm";
 
 import { db } from "@echo-webkom/db";
 import {
   happenings,
   happeningsToGroups,
   questions,
-  registrations,
   spotRanges,
   type HappeningInsert,
-  type HappeningsToGroupsInsert,
   type QuestionInsert,
-  type SpotRangeInsert,
 } from "@echo-webkom/db/schemas";
-import { type HappeningType } from "@echo-webkom/lib";
 
 import { withBasicAuth } from "@/lib/checks/with-basic-auth";
 import { isBoard } from "@/lib/is-board";
-import { client } from "@/sanity/client";
-import { happeningQuerySingle, type SanityHappening } from "./sync/query";
+import { toDateOrNull } from "@/utils/date";
+import { makeListUnique } from "@/utils/list";
+import { type SanityHappening } from "./sync/query";
 
 export const dynamic = "force-dynamic";
 
-const sanityPayloadSchema = z.object({
-  _id: z.string(),
-});
-
-const revalidate = (type: HappeningType, slug: string) => {
-  revalidatePath("/");
-  revalidatePath(`/${type === "bedpres" ? "bedpres" : "arrangement"}/${slug}`);
-};
-
 export const POST = withBasicAuth(async (req) => {
-  const startTime = new Date().getTime();
+  const { operation, documentId, data } = (await req.json()) as unknown as {
+    operation: "create" | "update" | "delete";
+    documentId: string;
+    data: SanityHappening | null;
+  };
 
-  const payload = sanityPayloadSchema.safeParse(await req.json());
+  // eslint-disable-next-line no-console
+  console.log(operation, documentId, JSON.stringify(data));
```

Comment:
console.log

Reply:
Ja. Vil ha console log her for å kunne debugge om noe skulle gå galt engang.